.ci/tools/go.mod

```diff
@@ -270,6 +272,7 @@ require (
 	github.com/timonwong/loggercheck v0.9.4 // indirect
 	github.com/tomarrell/wrapcheck/v2 v2.8.3 // indirect
 	github.com/tommy-muehle/go-mnd/v2 v2.5.1 // indirect
+	github.com/uber-go/gopatch v0.3.0 // indirect
```

Comment:
Pinned to `0.3.0` for now as the latest version (`0.4.0`) requires Go 1.22, which the provider has not yet upgraded to. Will open an issue reminder to upgrade this as part of the provider upgrade.